cmd/operator-sdk/test/local.go

```diff
@@ -279,10 +279,10 @@ func replaceImage(manifestPath, image string) error {
 		if err != nil {
 			return err
 		}
-		dep := &appsv1.Deployment{}
+		var dep appsv1.Deployment
 		switch o := obj.(type) {
 		case *appsv1.Deployment:
-			dep = o
+			dep = *o
```

Comment:
AH! :bulb: I get it now. We were creating an instance only to reassign it anyway.